tests/strainMeasurement/plotVelocities.py

```diff
@@ -0,0 +1,656 @@
+#!/usr/bin/env/python
+
+import sys,traceback
+if sys.version_info[0] != 3:
+    print("This script is only python3 compatible!")
+    exit(1)
+
+
+import numpy as np
+import h5py
+from attrdict import AttrMap,AttrDict
+from argparse import ArgumentParser
+import os.path
+import json
+
+# Import Plot Settings
+sys.path.append( os.environ["PYTHONSCRIPT_DIR"] )
```

Comment:
Sorry yes, wanted to change this on friday forgot,

lets see :-)


> Am 25.03.2016 um 19:08 schrieb Xylar Asay-Davis <notifications@github.com>:
> 
> In tests/strainMeasurement/plotVelocities.py <https://github.com/xylar/acciv/pull/6#discussion_r57471777>:
> 
> > +
> > +import sys,traceback
> > +if sys.version_info[0] != 3:
> > +    print("This script is only python3 compatible!")
> > +    exit(1)
> > +
> > +
> > +import numpy as np
> > +import h5py
> > +from attrdict import AttrMap,AttrDict
> > +from argparse import ArgumentParser
> > +import os.path
> > +import json
> > +
> > +# Import Plot Settings
> > +sys.path.append( os.environ["PYTHONSCRIPT_DIR"] )
> The PlotSettings package and the PYTHONSCRIPT_DIR seem to be something specific to your setup. At least I couldn't figure out what this is.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/xylar/acciv/pull/6/files/a8623767820dfafe808a01e24faa87c44c0a8c02#r57471777>